src/Simulation/Common/QubitManagerTrackingScope.cs

```diff
@@ -44,8 +44,8 @@ public List<Qubit> Locals
         private Stack<StackFrame> operationStack; // Stack of operation calls.
         private StackFrame curFrame; // Current stack frame - all qubits in current scope are listed here.
 
-        public QubitManagerTrackingScope(long qubitCapacity, bool mayExtendCapacity = false, bool disableBorrowing = false) 
-            : base(qubitCapacity, mayExtendCapacity, disableBorrowing)
+        public QubitManagerTrackingScope(long qubitCapacity, bool mayExtendCapacity = false, bool disableBorrowing = false, bool encourageReuse = true) 
```

Comment:
nit: as far as I know we don't have rules about page width in C# projects, but it's really inconvenient to work with or review wide code on a laptop... Any objections to staying within 120 chars width?

Reply:
Sure. Done.